core/cap-0046-09.md

```diff
@@ -110,28 +113,27 @@ have exactly the same upgrade set).
 Individual config settings are validated in the way defined by the CAPs
 introducing them.
 
-#### `ConfigUpgradeSet` acceptance
+#### Disabling Soroban Transactions
```

Comment:
Why did you remove the 'acceptance' section? I don't think it has changed much.

Reply:
That section was mainly about validators accepting a ConfigSettingEntry through some mechanism in consensus  in consensus. I didn't think it was necessary (or accurate) now that the entry is expected to live on ledger.

Reply:
Ok, I think we still need to mention somewhere that validators can only vote for an upgrade set key that they have scheduled (maybe not as a separate section).

Reply:
I mention this under the `Configuration upgrade mechanism` section. Do you think that's sufficient?
```
The actual `ConfigUpgradeSet` will need
to exist on ledger in a `ContractDataEntry`, and the validators will look it up
using the `ConfigUpgradeSetKey`.
```

Reply:
Ah ok. Updated in the last commit.